src/stake-pool/utils.ts

```diff
@@ -242,6 +242,27 @@ export async function calcWithdrawals(
   return res;
 }
 
+export async function calcWithdrawalsInverse(
+  withdrawalAmountLamports: Numberu64,
+  stakePoolAccount: StakePoolAccount,
+  validatorList: ValidatorList,
+): Promise<ValidatorWithdrawalReceipt[]> {
+  const {
+    account: { data: stakePool },
+  } = stakePoolAccount;
+  const { dropletsUnstaked } = await calcWithdrawalReceiptInverse(
+    withdrawalAmountLamports,
+    stakePool,
+  );
+  const res = await calcWithdrawals(
+    dropletsUnstaked,
+    stakePoolAccount,
+    validatorList,
+  );
+
+  return res;
```

Comment:
```suggestion
  return calcWithdrawals(
    dropletsUnstaked,
    stakePoolAccount,
    validatorList,
  );
```

Any reason for the additional await? Think we can just return the promise here?

Reply:
nope my bad, fixed it 😅